packages/grafana-runtime/src/utils/licensing.ts

```diff
@@ -1,6 +1,12 @@
+import { FeatureToggles } from '@grafana/data';
 import { config } from '../config';
 
-export const featureEnabled = (feature: string): boolean => {
-  const { enabledFeatures } = config.licenseInfo;
-  return enabledFeatures && enabledFeatures[feature];
+export const featureEnabled = (feature: boolean | undefined | keyof FeatureToggles): boolean => {
```

Comment:
@DanCech -- this now uses the same `featureToggle` bucket, but note that the license is checked on the backend before anythign can be enabled